What's new 8.10 (#3901)
elastic/security-docs: docs/whats-new.asciidoc

// tag::notable-highlights[]

[float]
== Navigation menu updates

The Security navigation menu has been updated with reorganized sections and a refreshed design. In addition, a new **Rules** section allows you to access the following pages:

* Rules
* Benchmark Integrations
* Shared Exception Lists
* MITRE ATT&CK® coverage

[role="screenshot"]
image::whats-new/images/8.10/nav-overview.gif[Security navigation menu]

[float]
== Elastic AI Assistant enhancements 

A new RBAC setting controls user access to the {security-guide}/security-assistant.html[Elastic AI Assistant].

[role="screenshot"]
image::whats-new/images/8.10/ai-assistant-privilege.png[Elastic AI Assistant Kibana privilege]

[float]
== Detection rules and alerts enhancements

[float]
=== MITRE ATT&CK® coverage page

The {security-guide}/rules-coverage.html[MITRE ATT&CK® coverage page] shows which MITRE ATT&CK® adversary tactics and techniques are covered by your installed and enabled detection rules. This includes both Elastic prebuilt rules and custom rules.

[role="screenshot"]
image::whats-new/images/8.10/rules-coverage.png[MITRE ATT&CK® coverage page]

[float]
=== New prebuilt rule details flyout

> The new prebuilt rule details flyout allows you to examine the details of a prebuilt rule before you {security-guide}/prebuilt-rules-management.html[install or update] it. You can access this flyout by clicking a rule name on either the **Add Elastic Rules** page or the **Rule updates** table. The flyout displays the About, Definition, and Schedule sections, as shown on the rule details page. It also shows the setup and investigation guides for rules that have them.

Reviewer: ```suggestion
The new prebuilt rule details flyout allows you to examine the details of a prebuilt rule before you {security-guide}/prebuilt-rules-management.html[install or update] it. You can access this flyout by clicking a rule name on the **Add Elastic Rules** page or the **Rule updates** table. The flyout displays the **About**, **Definition**, and **Schedule** sections, as shown on the rule details page. It also shows the setup and investigation guides for rules that have them.
```

 
[role="screenshot"]
image::whats-new/images/8.10/prebuilt-rule-details-flyout.png[Prebuilt rule details flyout]

[float]
=== Enhanced alert details flyout UI

> The redesigned alert details experience presents relevant context, insights, and more, while investigating an alert. Use the collapsed view to access summarized information, and then expand each section to open detailed views. Additional improvements include:

Reviewer: ```suggestion
The redesigned alert details experience presents relevant context and insights while investigating an alert. Use the collapsed view to access summarized information, and then expand each section to open detailed views. Additional improvements include:
```

> The redesigned alert details experience presents relevant context, insights, and more, while investigating an alert. Use the collapsed view to access summarized information, and then expand each section further to open detailed views. Additional improvements include:

Reviewer 2: Nit: This describes expanding it just once, not expanding it and then expanding it again.
```suggestion
The redesigned alert details experience presents relevant context, insights, and more, while investigating an alert. Use the collapsed view to access summarized information, and then expand each section to open detailed views. Additional improvements include:
```

[role="screenshot"]
image::whats-new/images/8.10/custom-highlighted-fields.png[Custom highlighted fields]

[float]
== Reputation service

When configuring {security-guide}/configure-endpoint-integration-policy.html#behavior-protection[malicious behavior protection] on an {elastic-defend} policy, you can now select to use **Reputation service**. This service identifies malicious activity and false positives, and enriches alerts using data from various sources, such as VirusTotal and telemetry. For example, reputation service can detect suspicious downloads of binaries with low or malicious reputation.

NOTE: Reputation service requires an active https://www.elastic.co/pricing[Platinum or Enterprise subscription] and is available on cloud deployments only.

[float]
== Cloud Security enhancements

[float]
=== Organization-wide onboarding for cloud security posture management on AWS

> This release automates the onboarding of every account within an AWS Organization to cloud security posture management (CSPM) — including existing and new accounts. With {security-guide}/cspm-get-started.html#cspm-setup[AWS CloudFormation], onboarding takes just a few clicks. This helps you quickly get a comprehensive view of the security posture of all your current and future AWS accounts.

Reviewer: ```suggestion
This release automates the onboarding of every AWS Organization account to cloud security posture management (CSPM) — including existing and new accounts. With {security-guide}/cspm-get-started.html#cspm-setup[AWS CloudFormation], onboarding takes just a few clicks. This helps you quickly get a comprehensive view of the security posture of all your current and future AWS accounts.
```

> This release automates the onboarding of every account within an AWS Organization to cloud security posture management (CSPM) — including existing and new accounts. With {security-guide}/cspm-get-started.html#cspm-setup[AWS CloudFormation], onboarding takes just a few clicks. This helps you quickly get a comprehensive view of the security posture of all your AWS accounts — both current and future.

Reviewer 2: Suggested edit to mix up repetitive sentence structure:
```suggestion
This release automates the onboarding of every account within an AWS Organization to cloud security posture management (CSPM) — including existing and new accounts. With {security-guide}/cspm-get-started.html#cspm-setup[AWS CloudFormation], onboarding takes just a few clicks. This helps you quickly get a comprehensive view of the security posture of all your current and future AWS accounts.
```

[float]
=== Cloud security posture management, now for Google Cloud

> {elastic-sec} 8.10 expands its cloud security posture management (CSPM) capabilities to cover {security-guide}/cspm-get-started-gcp.html[Google Cloud]. You can now assess and bolster the security posture of your GCP assets right from our platform.

Reviewer: ```suggestion
Cloud security posture management (CSPM) capabilities have been expanded to cover {security-guide}/cspm-get-started-gcp.html[Google Cloud]. You can now assess and bolster the security posture of your GCP assets right from our platform.
```

[role="screenshot"]
image::whats-new/images/8.10/ai-assistant-privilege.png[Elastic AI Assistant Kibana privilege]

[float]
== Detection rules and alerts enhancements

[float]
=== MITRE ATT&CK® coverage page

The {security-guide}/rules-coverage.html[MITRE ATT&CK® coverage page] shows which MITRE ATT&CK® adversary tactics and techniques are covered by your installed and enabled detection rules. This includes both Elastic prebuilt rules and custom rules.

[role="screenshot"]
image::whats-new/images/8.10/rules-coverage.png[MITRE ATT&CK® coverage page]

[float]
=== New prebuilt rule details flyout

The new prebuilt rule details flyout allows you to examine the details of a prebuilt rule before you {security-guide}/prebuilt-rules-management.html[install or update] it. You can access this flyout by clicking a rule name on either the **Add Elastic Rules** page or the **Rule updates** table. The flyout displays the About, Definition, and Schedule sections, as shown on the rule details page. It also shows the setup and investigation guides for rules that have them.
 
[role="screenshot"]
image::whats-new/images/8.10/prebuilt-rule-details-flyout.png[Prebuilt rule details flyout]

[float]

> === Redesigned alert details flyout

Reviewer 3: ```suggestion
=== Enhanced alert details flyout UI
```

> To be added

Reviewer 3: ```suggestion
The redesigned alert details flyout gives you more options and flexibility when investigating an alert. Check the sections in the right panel to get a summary of important alert details. Analyze each section further by opening their detailed views. Additional core improvements include:

* Previews of rule details and visualizations allow you to stay within the flyout when investigating the alert.
* Investigation guides are easier to find and read.
* Alert insights now include prevalence information on related hosts and users.

```

Reviewer 3: @jmikell821 you can reuse the [gif](https://www.elastic.co/guide/en/security/master/images/open-alert-details-flyout.gif) I created for the alert details docs. The file is `open-alert-details-flyout.gif` and it's located at: `docs/detections/images/open-alert-details-flyout.gif`

Reviewer 3: Changes @paulewing suggested via Slack:

```suggestion
The redesigned alert details experience presents relevant context, insights, and more while investigating an alert. Utilize the collapsed view to see summarized information and then expand each section further to open detailed views. Additional improvements include:

* Previews of rule details and visualizations allow you to stay within the flyout when investigating the alert.
* Investigation guides are easier to find and read.
* Alert insights now include prevalence information on related hosts and users.
```

NOTE: Reputation service requires an active https://www.elastic.co/pricing[Platinum or Enterprise subscription] and is available on cloud deployments only.

[float]
== Cloud Security enhancements

[float]
=== Organization-wide onboarding for cloud security posture management on AWS

This release automates the onboarding of every account within an AWS Organization to cloud security posture management (CSPM) — including existing and new accounts. With {security-guide}/cspm-get-started.html#cspm-setup[AWS CloudFormation], onboarding takes just a few clicks. This helps you quickly get a comprehensive view of the security posture of all your AWS accounts — both current and future.

[float]
=== Cloud security posture management, now for Google Cloud

{elastic-sec} 8.10 expands its cloud security posture management (CSPM) capabilities to cover Google Cloud. You can now assess and bolster the security posture of your GCP assets right from our platform.

> // link to be added when PR #3809 is merged {security-guide}/cspm-get-started-gcp.html.html[Google Cloud].

Author: ```suggestion
// link to be added when PR #3806 is merged {security-guide}/cspm-get-started-gcp.html.html[Google Cloud].
```
